addons/mail/static/src/discuss/call/common/rtc_session_model.js

```diff
@@ -27,8 +27,10 @@ export class RtcSession {
     /** @type {Date|undefined} */
     raisingHand;
     videoComponentCount = 0;
-    /** @type {MediaStream} */
-    videoStream;
+    /** @type {MediaStream{}}} */
```

Comment:
There is one `}` that is not needed :)
That being said, this type doesn't look right. I guess you wan't to says something like "An Object with multiple MediaStream". I that case, you have to specify the type of the key, eg:

```javascript
    /** @type {Object.<number, MediaStream>} */
```
Note: `number` is an example, I don't know what is used.